make/tests

```diff
@@ -2,24 +2,11 @@
 # Build test executables
 ##
 
-ifeq ($(OS),Windows_NT)
-LIB_DIRS_FULL=$(filter %/, $(wildcard src/test/interface/*/) src/test/interface/)
-LIB_DIRS_TBB=$(LIB_DIRS_FULL:src/%/=%/tbb.dll)
-else
-LIB_DIRS_TBB=
-endif
-
-test/%/tbb.dll : $(TBB_TARGETS)
-	@mkdir -p test/$*
-	cp -v $(TBB_TARGETS) test/$*
-
-.PRECIOUS: $(LIB_DIRS_TBB)
```

Comment:
All of this was here so that Windows Jenkins tests worked, local tests are fine. A nicer way of doing this is building math-libs first on Jenkins. See below.